objax/nn/layers.py

```diff
@@ -327,6 +327,63 @@ def __call__(self, x: JaxArray) -> JaxArray:
         self.avg.value += (self.avg.value - x) * (self.momentum - 1)
         return self.avg.value
 
+class RNN(Module):
```

Comment:
I think the name `RNN` is too generic.
Pretty much any type of recurrent block (LSTM, GRU, ....) could be called RNN.
Is there some better way to call it?

Reply:
Also RNN refers to the architecture, not to the cell. Here's what TF/Keras does https://www.tensorflow.org/api_docs/python/tf/compat/v1/nn/rnn_cell/RNNCell

Reply:
This is a specific RNN architecture that operates across time (so not a cell).  I would call this something like SimpleRNN; and make sure it replicates keras' SimpleRNN functionality with default arguments:

https://www.tensorflow.org/api_docs/python/tf/keras/layers/SimpleRNN

RNN you could reserve as an object that takes an RNNCell and performs a scan across time.

Reply:
Changed the name to SimpleRNN